src/Microsoft.ML.Transforms/Text/TextCatalog.cs

```diff
@@ -496,8 +496,8 @@ public static class TextCatalog
              => new NgramHashingEstimator(Contracts.CheckRef(catalog, nameof(catalog)).GetEnvironment(), columns);
 
         /// <summary>
-        /// Uses <a href="https://arxiv.org/abs/1412.1576">LightLDA</a> to transform a document (represented as a vector of floats)
-        /// into a vector of floats over a set of topics.
+        /// Create a <see cref="LatentDirichletAllocationEstimator"/>, which uses <a href="https://arxiv.org/abs/1412.1576">LightLDA</a> to transform text (represented as a vector of floats)
+        /// into a vector of floats indicating the similarity of the text with each topic identified.
```

Comment:
>floats [](start = 29, length = 6)

single

Reply:
Should this also be crefd?

---
In reply to: [277098467](https://github.com/dotnet/machinelearning/pull/3442#discussion_r277098467) [](ancestors = 277098467)